backend/chainlit/config.py

```diff
@@ -251,6 +276,23 @@ def init_config(log=False):
     elif log:
         logger.info(f"Config file already exists at {config_file}")
 
+    if not os.path.exists(config_translation_dir):
+        os.makedirs(config_translation_dir, exist_ok=True)
+        logger.info(
+            f"Created default translation directory at {config_translation_dir}"
+        )
+
+    for file in os.listdir(TRANSLATIONS_DIR):
+        if file.endswith(".json"):
+            src = os.path.join(TRANSLATIONS_DIR, file)
+            dst = os.path.join(config_translation_dir, file)
+            with open(src, "r", encoding="utf-8") as f:
+                translation = json.load(f)
+                if not os.path.exists(dst):
```

Comment:
this condition can be moved above the with open to avoid opening files for nothing

Reply:
Perfect, @willydouhard . Improved in the next commit